tools/ctor_evaller.py

```diff
@@ -5,8 +5,11 @@
 '''
 
 import os, sys, json, subprocess, time
-import shared, js_optimizer
-from tempfiles import try_delete
+
+sys.path.insert(1, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
why is this needed?

Reply:
This is to access `tools` package correctly by adding proper system path. `emcc.py` can access `tools` because the root path is already automatically added to system path, but not in this case as `ctor_evaller` is inside `tools` directory.

It cannot access the package by `from . import shared` because 1) `from .` does not load any package 2) so is only allowed to package internal modules not scripts.

Related SO post: https://stackoverflow.com/questions/11536764/how-to-fix-attempted-relative-import-in-non-package-even-with-init-py

Reply:
`import` on Python 2 is a magic and it does all the path and relative import things implicitly. Not in Python 3.

Reply:
https://github.com/kripken/emscripten/blob/incoming/tests/runner.py#L25

An existing example from the current code.

Reply:
Got it, thanks.